services/brig/src/Brig/API/User.hs

```diff
@@ -250,14 +251,8 @@ createUserSpar new = do
 
   pure $! CreateUserResult account Nothing Nothing (Just userTeam)
   where
-    lmap f =
-      let h = \case
-            Left x ->
-              Left (f x)
-            Right a ->
-              Right a
-          {-# INLINE h #-}
-       in ExceptT . fmap h . runExceptT
+    lmap :: Functor m => (a -> b) -> ExceptT a m v -> ExceptT b m v
+    lmap f = ExceptT . fmap (mapLeft f) . runExceptT
```

Comment:
This looks like [withExceptT](https://hoogle.zinfra.io/file/root/.ghcup/ghc/8.10.7/share/doc/ghc-8.10.7/html/libraries/transformers-0.5.6.2/Control-Monad-Trans-Except.html#v:withExceptT).